lib/cli.coffee

```diff
@@ -657,17 +659,43 @@ switch command
               exit err if err
 
               # insert pages
-              async.forEach objects, ((object, next) ->
+              async.map objects, ((object, next) ->
                 data = object.val()
+                old_id = data._id
                 delete data._id
                 data.seo = {}
                 data.site_id = dst_site.get('_id').val()
-                db.get('objects').insert data, (err) ->
+                db.get('objects').insert data, (err, new_doc) ->
+                  id_mapping[old_id] = new_doc.val()._id
+                  next null, new_doc
                   exit err if err
-                  next()
-              ), (err) ->
-                exit err if err
-                exit 'clone complete'
+              ), (err, objects_cloned) ->
+
+                # clean up references
+                async.forEach objects_cloned, ((object, next) ->
+                  object_val = object.val()
+                  str_json = JSON.stringify(object_val, null, 3)
+                  replaced = false
+
+                  # replace mapping
+                  for key, value of id_mapping
+                    if str_json.indexOf(key) > -1
+                      og_key = new RegExp(key, 'g');
+                      replaced = true
+                      str_json = str_json.replace(og_key, value)
+
+                  if replaced
+                    new_json = JSON.parse(str_json)
+
+                    # set new data with updated mapping
+                    object.get('data').set new_json.data, (err) ->
```

Comment:
don't push anymore because i'm in the middle of updates, but before you did `object.val()`, but now you're setting to `object.get('data').val()`